examples/README.md

```diff
@@ -69,47 +69,49 @@ with the lane owner.
     |       |       |       |X|       |       |      |
     |       |       |       |X|       |       |      |
 
+Every turn the driver moves one step forward and gains 10 points
+if there was no obstacle.  
 Here are the possible obstacles and the action you should take when hitting
 them:
 
 * `obstacles.NONE`      there is no obstacle at this position. You can return any
                         action.
                         Score : 0
 
-* `obstacles.PENGUIN`   return `actions.PICKUP` and move 1 step forward. You will
-                        not move backward if you do not pick up the penguin.
+* `obstacles.PENGUIN`   return `actions.PICKUP` and move forward. You will
+                        move forward even if you do not pick up the penguin.
```

Comment:
Why did you remove "1 step forward"? Getting extra points and movement is intended. This is a game about Linux, so picking up penguins should be special.

Having obstacles that have higher priority makes the game more interesting and challenging. For example if you have 2 obstacles you can handle in future move, player that give priority to the penguin should be more successful.

Reply:
This change actually makes it. Before picking up the penguin would have given you 20 points and 2 steps. The same as just moving forward.

Moreover, there were situations where the extra move puts you on an obstacle and you 'lose' points since you don't get the chance to handle it.

Reply:
Another option is keeping the extra step, clearing the field after the penguin, adding 20 points for forward (2 steps) + adding 10 points for penguin pickup +? adding points for removed obstacle (can be another penguin).

In any case, keeping the current score doesn't give the advantage for picking up the penguin.